cms/djangoapps/contentstore/views/tests/test_assets.py

```diff
@@ -86,6 +88,42 @@ def test_pdf_asset(self):
         # Note: Actual contentType for textbook.pdf in asset.json is 'text/pdf'
         self.assertEqual(content.content_type, 'application/pdf')
 
+    def test_relative_url_for_split_course(self):
+        """
+        Test relative path for split courses assets
+        """
+        with modulestore().default_store(ModuleStoreEnum.Type.split):
+            module_store = modulestore()
+            course_id = module_store.make_course_key('edX', 'toy', '2012_Fall')
+            import_course_from_xml(
+                module_store,
+                self.user.id,
+                TEST_DATA_DIR,
+                ['toy'],
+                static_content_store=contentstore(),
+                target_id=course_id,
+                create_if_not_present=True
+            )
+            course = module_store.get_course(course_id)
+
+            html_src_attribute = '"/static/sample_static.txt"'
+            filename = 'sample_static.txt'
+            asset_url = replace_static_urls(html_src_attribute, course_id=course.id)
+            url = asset_url.replace('"', '')
```

Comment:
because the src attr is in quotes like ```<script src="aaaaa"></script>``` and replace_static_urls used in regex to determine the asset link [here](https://github.com/edx/edx-platform/blob/212598e649bb358921f8dee4f930eef81af531ff/common/djangoapps/static_replace/__init__.py#L24)